### DISCUSSION TITLE
the inside modal's dropdown is automaticly opening as you open the modal

### DISCUSSION THREAD

**Discussion by ramonamg17** [fomantic/Fomantic-UI#1427] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Bug Report
the  modal's inside dropdown is automaticly opening as you open the modal

## Steps to reproduce
1.
2.
3.

## Screenshot (when possible)
![]()

## Version
x.x.x

**Comment by dutrieux** [comment, marked as answer]
You can set `autofocus`  to `false` on setting of modal : https://jsfiddle.net/dutrieux/wvgo5ty2/

**Comment by lubber-de** [comment]
Closing, because that's what `autofocus` is for as @dutrieux pointed out.
